coordinator.lic

```diff
@@ -0,0 +1,478 @@
+custom_require.call(%w[common drinfomon common-travel common-items common-arcana common-money])
+
+class Coordinator
+  include DRC
```

Comment:
💡 Why is `common-items` in the `custom_require` list but not the `include` list? Does it need to be here or removed from above?